src/components/cluster/detail/__tests__/PortMappingsRow.js

```diff
@@ -0,0 +1,49 @@
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { ThemeProvider } from 'emotion-theming';
+import React from 'react';
+import theme from 'styles/theme';
+
+import PortMappingsRow from '../PortMappingsRow';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  render(
+    <Router>
+      <ThemeProvider theme={theme}>
+        <PortMappingsRow />
+      </ThemeProvider>
+    </Router>,
+    div
```

Comment:
According to [the docs](https://testing-library.com/docs/react-testing-library/api#render),  you'd have to provide the container in a different way, but also remember you'd still have to append the newly created `div` to the DOM for it to actually get rendered (there's an example on the docs link).

Reply:
Yeah this is how you would use the regular react `render`.
It works because `react-testing-library``render` uses react render under the hood I think, so if you don't wanna make queries, I would say you can totally use it like Oliver does here.

Reply:
The way it's used here is the same as just using `render(<Component />)` (which also works), the second argument of the function should be an options object (you can see that in the docs). If you really want to render in a container you'd provide a second argument similar to:
```js
  render(<Component />, {
    container: containerElement
  }
```

But again, to actually render in a container element, that container element should be in the DOM. Otherwise, nothing happens.

Reply:
Citing the `testing-library` docs "By default, React Testing Library will create a div and append that div to the document.body and this is where your React component will be rendered. If you provide your own HTMLElement container via this option, it will not be appended to the document.body automatically."

Reply:
Not sure what the consequence / impact is of how it's done right now.

If we should change it, then lets change it everywhere, because I think I copied it from another test.

Will do some more reading later, and consider it out of scope for this PR, but certainly something to align if it's wrong.

Reply:
Im surprised it works then...

Will consider it out of scope of this PR. If we're doing it wrong then lets fix it everywhere at once. Have to do a bit of reading to understand this a bit beter.